layers/+themes/dark-window/packages.el

```diff
@@ -0,0 +1,33 @@
+;;; packages.el --- dark-window layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Oleg Pykhalov <go.wigust@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst dark-window-packages
+  '(frame-cmds))
+
+(defun dark-window/init-frame-cmds ()
+  (use-package frame-cmds
+    :config
+    ;; Based on Nicolas Petton code http://nicolas.petton.fr/blog/emacs-dark-window-decoration.html
+    (progn
+      (defun set-selected-frame-color ()
```

Comment:
Need to find a better name. The function doesn't set an arbitrary color, it specifically sets the decoration to dark. In fact, we'd probably want an argument to decide if the decoration should set to dark or light.